platform/viewer/src/connectedComponents/ConnectedHeader.js

```diff
@@ -1,8 +1,13 @@
 import Header from '../components/Header/Header.js';
 import { connect } from 'react-redux';
 
+const isActive = a => a.active === true;
+
 const mapStateToProps = state => {
+  const activeServer = state.servers.servers.find(isActive);
+
   return {
+    server: activeServer,
```

Comment:
With the Header changes I'm suggesting, this could be removed.

Reply:
I will remove it...